transcom/mymove
# [MB-16989] Label/prefix service item dates

## [MB-16989](https://dp3.atlassian.net/browse/MB-16989)

## Summary

Adds clarity to the date displayed alongside MTO service items

Remaining before removing from draft state:
- [ ] Refine MB-16989 and prepare for sprint
- [x] Add unit tests confirming concatenation of prefix and date to `RequestedServiceItemTable.test.jsx`

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1. Create an MTO shipment with several service items. There are several SIT related test harness scenarios that can quickly accomplish this
2. Approve and reject some as the TOO to see all 3 statuses simultaneously, updating in real time

Alternatively, use Storybook:

1. `yarn storybook`
2. go to http://localhost:6006/?path=/story/office-components-requestedserviceitemstable--default

### Frontend


## Screenshots

Requested:
<img width="1052" alt="image" src="https://github.com/transcom/mymove/assets/15805554/4a5aebce-168e-4e63-9ca9-593d8e9d70ea">

Approved:
<img width="1049" alt="image" src="https://github.com/transcom/mymove/assets/15805554/5ca3afc7-eb3e-4eb1-ae67-92f8293ca9d6">

Rejected:
<img width="1049" alt="image" src="https://github.com/transcom/mymove/assets/15805554/5f470856-a63b-499f-839e-f01fa6bb3c89">


[MB-16989]: https://dp3.atlassian.net/browse/MB-16989?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
